### PR TITLE
BRS-1094: Moving activity related code to ActivityService

### PR DESCRIPTION
**Pull request by danieltruong**
### Jira Ticket:
BRS-1094

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/BRS-1094

### Description:
Moving activity related service to ActivityService.

NOTE: This must be deployed at the same time as https://github.com/bcgov/bcparks-ar-api/pull/123

### CODE REVIEW

File: src/app/enter-data/sub-area-search/sub-area-search.component.ts [hunk 1]
```diff
@@ -65,11 +64,15 @@ export class SubAreaSearchComponent implements OnDestroy {
     );
   }
 
+  ngAfterViewInit() {
```
Comment (marklise) [inline src/app/enter-data/sub-area-search/sub-area-search.component.ts:67]: If keeping here - `impements AfterViewInit` required @ constructor.

File: src/app/enter-data/sub-area-search/sub-area-search.component.ts [hunk 2]
```diff
@@ -65,11 +64,15 @@ export class SubAreaSearchComponent implements OnDestroy {
...
+
   datePickerOutput(event) {
-    console.log("***********************************************event:", event);
-    console.log("1", new Date(this.modelDate));
-    console.log("2", this.maxDate);
-    console.log("3", new Date(this.modelDate) <= this.maxDate);
+    console.log('***********************************************event:', event);
+    console.log('1', new Date(this.modelDate));
+    console.log('2', this.maxDate);
+    console.log('3', new Date(this.modelDate) <= this.maxDate);
```
Comment (marklise) [inline src/app/enter-data/sub-area-search/sub-area-search.component.ts:75]: Toss these

Comment (danieltruong) [reply src/app/enter-data/sub-area-search/sub-area-search.component.ts]: [e6b8b99](https://github.com/bcgov/bcparks-ar-admin/pull/220/commits/e6b8b99334f7447fe6129101c7c624b337928e29)

File: src/app/enter-data/sub-area-search/sub-area-search.component.ts (hunk 1 above)
Comment (danieltruong) [reply src/app/enter-data/sub-area-search/sub-area-search.component.ts:67]: [e6b8b99](https://github.com/bcgov/bcparks-ar-admin/pull/220/commits/e6b8b99334f7447fe6129101c7c624b337928e29)

**Review by marklise — APPROVED** [review]

**Review by cameronpettit — APPROVED** [review]

**Pull request merged**